packages/svelte/src/compiler/phases/3-transform/client/visitors/template.js

```diff
@@ -2070,12 +2071,11 @@ export const template_visitors = {
 			}
 		};
 
+		namespace = node.metadata.svg ? namespace_svg : null;
+
 		for (const attribute of node.attributes) {
 			if (attribute.type === 'Attribute') {
 				attributes.push(attribute);
-				if (attribute.name === 'xmlns' && is_text_attribute(attribute)) {
-					namespace = attribute.value[0].data;
-				}
```

Comment:
I think it would be good to add this to the analysis phase of `SvelteElement`: check if there's such an attribute, and if yes, if it's refering to the SVG namespace